test/common/controllers/base-address.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const proxyquire = require('proxyquire');
+class StubBaseController {}
+
+StubBaseController.prototype.locals = sinon.stub();
+
+const BaseAddressController = proxyquire('../../../apps/common/controllers/base-address', {
+  './base': StubBaseController
+});
+
+describe('BaseAddressController', function() {
+
+  it('extends base-controller', ()=> {
+    BaseAddressController.prototype.should.be.an.instanceOf(StubBaseController);
+  });
+
+  it('has a locals, translateCategories, mapAddress & hasCategories methods', ()=> {
+    BaseAddressController.prototype.should.have.property('locals').that.is.a('function');
```

Comment:
"one assertion per test" is IMO entirely analogous to "one line of code per function". 

Use as many assertions as are needed to completely test the functionality described by the test. Certainly having *loads* of assertions is severe code smell, and can indicate the scope of the test is too broad, but if the test is readable and understandable (like this one) then it's totally fine.